packages/browser-wallet/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 1.1.1
+
+### Fixed
+
+-   Support number properties as bigints for verifiable credentials.
+-   Use localization for verifiable credentials.
+-   Fix wallet crashing when importing verifiable credentials with new schemas.
```

Comment:
What was the fix for this? I can't identify any changes pertaining to schema's when importing, assuming you, by "schemas" mean the credential schemas stored?

Reply:
OK, thanks for clarifying 🙂